src/EnergyPlus/ZoneEquipmentManager.hh

```diff
@@ -57,6 +57,7 @@
 #include <EnergyPlus/DataGlobalConstants.hh>
 #include <EnergyPlus/DataGlobals.hh>
 #include <EnergyPlus/DataHeatBalance.hh>
+#include <EnergyPlus/DataSizing.hh>
```

Comment:
@Myoldmopar I was looking at this line and thought, ZoneEquipmentManager does zone sizing so why did I need to add this include? Well, because of the change to enum for the DOASControl var (line 145 below) that header is needed here. OK, so I can delete the include in the cc then, right? Yes, that was fine. Then I thought that I could delete all these includes here in the hh from the cc. Nope, that did not work. #include DataHeatBalance.hh needs to be in both cc and hh, at least as far as the compiler is concerned. That doesn't make sense to me so I am hesitant to push up a change to the #includes in this branch.

Reply:
So in general, the preferred pattern would be to keep headers themselves as light as possible.  Only include what is absolutely necessary for the header code to compile.  And in a CC file, explicitly list whatever headers are necessary, don't rely on a different included file to include what you need.  Keeping the headers as light as possible could aid in compile time, because each header is `#include`d in so many other files.  Does that make sense?

Reply:
I just pushed the conflict update. Also noticed that #include DataZoneEnergyDemands.hh (the conflicting line) was here and in the cc, so I removed from the cc. I checked and all these includes in the hh are needed for the definitions here. Those same includes are not needed in the cc.

Reply:
I think there could be a dedicated small effort to clean up the headers being included but it's not worth investigating here.  Let's move on.